app-tests/next-layout-admin/src/pages/index.js

```diff
@@ -2,24 +2,24 @@ import '@espressive/legacy-css';
 import Head from 'next/head';
 import pkg from '../../package';
 
-import { Admin } from '@espressive/cascara';
+import { AdminStructure } from '@espressive/cascara';
 
 const Home = () => {
   return (
-    <Admin
+    <AdminStructure
       header={
-        <Admin.Header>
+        <AdminStructure.Header>
           <h1>{pkg.name}</h1>
-        </Admin.Header>
+        </AdminStructure.Header>
```

Comment:
If we are updating this example, we should also make sure that it follows the correct APIs as AdminStructure slot components. The h1 here as a child is not needed. Instead `pkg.name` can be passed as a prop to `AdminStructure.Header` not as a child. And the `h1` tag is not needed at all.